samples/minimal/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("io.element.android-compose-application")
```

Comment:
This run configuration was selected by default when opening the project, it will confuse a lot of people. Is there a way to make `app` the default configuration?

Reply:
I don't think so, it's local option, so can't do much about that

Reply:
I trying to see in the .idea folder, but we are ignoring the vast majority of those file. At least `app` and `minimal` should be documented somewhere.